http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/KubernetesServiceInterp.scala

```diff
@@ -96,6 +96,7 @@ class LeoKubernetesServiceInterp[F[_]: Parallel](
           googleProject,
           userInfo,
           petSA,
+          FormattedBy.Galaxy,
```

Comment:
why are we setting `FormattedBy` here? shouldn't we only update this when we know that the disk has been formatted ie when during monitoring we see that the runtime was created successfully? otherwise we might be setting it too early and during the startup script, it will think the disk is already formatted